src/config/dynamic-theme-fixes.config

```diff
@@ -12821,6 +12821,11 @@ CSS
 
 twitter.com
 
+INVERT
+[aria-label="Seek slider"]
+[aria-label="Volume slider"]
+[style^="flex-grow"]
```

Comment:
Hmm, well it seems good it could also be tricky to have it included. But it should be good for now.

Reply:
Yeah, it's a trick corner situation for sure. I couldn't find a better way to do it, so I'd be very interested in how else it could be fixed.

Reply:
> I couldn't find a better way to do it, so I'd be very interested in how else it could be fixed.

Nor can I so let's leave it for now and see if it breaks things.